Sources/Chat/ProtocolServices/Common/ResubscriptionService.swift

```diff
@@ -32,6 +32,7 @@ class ResubscriptionService {
 
     func resubscribe(account: Account) async throws {
         let topics = chatStorage.getThreads(account: account).map { $0.topic }
+        guard !topics.isEmpty else { return }
```

Comment:
may be do this check inside butchSubscribe?

Reply:
we do but it throws an error batchContainsNoItems